src/prefabs/pageWithOverviewAndDetials.tsx

```diff
@@ -1449,6 +1449,48 @@ const beforeCreate = ({
     return returnObject;
   };
 
+  const disabledKinds = createBlacklist([
```

Comment:
Login token, password should definitely not be whitelisted here. Object property as well because it doesn't work